docs/content/preview/yugabyte-voyager/migrate/assess-migration.md

```diff
@@ -161,7 +155,7 @@ A sample Migration Assessment report for PostgreSQL is as follows:
 
     1. **Without source database connectivity**: In situations where direct access to the source database is restricted, there is an alternative approach. Voyager includes packages with scripts for both PostgreSQL and Oracle present at `/etc/yb-voyager/gather-assessment-metadata`. You can perform the following steps for PostgreSQL with these scripts (similar for Oracle).
```

Comment:
For the "without source database connectivity" part, can we mention "[only PostgreSQL]"